### PR TITLE
fix github page

### PR DESCRIPTION
**Pull request by SirM2z**
Need to add a cname file after build

### CODE REVIEW

File: scripts/pullModels.js [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
 const { modulesOSSUrl } = require('./config');
- 
+
 (async () => {
   Object.keys(modulesOSSUrl).forEach(item => {
     modulesOSSUrl[item].forEach(itm => {
       download(itm, `dist/static/models/${item}`);
     });
   });
+  fs.writeFile(path.resolve(process.cwd(), './dist/CNAME'), 'pipboard.imgcook.com', (err) => {
```
Comment (yorkie) [inline scripts/pullModels.js]: This file is called pullModels, where we shouldn't do write this :)

**Review by yorkie — CHANGES_REQUESTED** [review]
It's good to maintain a CNAME file, and just copy that when building ghpage.

File: scripts/pullModels.js (hunk 1 above)
Comment (SirM2z) [reply scripts/pullModels.js]: good suggestion

**Comment by SirM2z** [comment]
Already modified :p @yorkie

**Review by yorkie — APPROVED** [review]
Looks good to me :)

**Pull request merged**